_topics/en/multisignature.md

```diff
@@ -0,0 +1,108 @@
+  |-|-|-|-|-|
+  | Multisig | `m` | `1` | `k` where `k<=m` | `k` | Uses `OP_CHECK*SIG` opcodes |
+  | Multisignature | `m` | `1` | 1 | `m` | Indistinguishable onchain from single-sig |
+  | Threshold signature | `m` | `1` | `1` | `k` where `k<=m` | Indistinguishable onchain from single-sig |
+  | Signature aggregation | `m` | `n` | 1 | `0` or `m`<br>(varies by scheme) | Proposed to allow a tx with multiple inputs to have only one signature, among other uses |
+
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+#primary_sources:
+#    - title: Test
+#    - title: Example
+#      link: https://example.com
+
+## ##################################################################### ##
+## NOTE: don't include things specific to MuSig; put those in the MuSig  ##
+## topic, which we link to in the see_also section                       ##
+## ##################################################################### ##
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: Discussion of multisignatures and threshold signatures
+    url: /en/newsletters/2020/07/01/#schnorr-signatures-and-multisignatures
+
+  - title: Mitigating power analysis attacks, including against multisignature schemes
+    url: /en/newsletters/2020/04/01/#mitigating-differential-power-analysis-in-schnorr-signatures
+
+  - title: Discussion about nested and composable multisignature schemes
+    url: /en/newsletters/2019/12/04/#continued-schnorr-taproot-discussion
+
+  - title: "BIPs #876 assigns BIP340 to multisignature compatible scheme"
+    url: /en/newsletters/2020/01/29/#bip340
+
+  - title: ECDSA multisignatures for scriptless Lightning Network payment channels
+    url: /en/newsletters/2018/10/09/#multiparty-ecdsa-for-scriptless-lightning-network-payment-channels
+
+  - title: Alternative to ECDSA multisignatures for signature adaptors
+    url: /en/newsletters/2020/04/08/#work-on-ptlcs-for-ln-using-simplified-ecdsa-adaptor-signatures
+
+  - title: Warning about using 160-bit addresses for naïve multiparty multisignatures
+    url: /en/newsletters/2020/06/24/#reminder-about-collision-attack-risks-on-two-party-ecdsa
+
+  - title: Implementing statechains for ECDSA using multisignatures
+    url: /en/newsletters/2020/04/01/#implementing-statechains-without-schnorr-or-eltoo
+
+  - title: Two papers published on fast ECDSA multisignatures
+    url: /en/newsletters/2018/10/23/#two-papers-published-on-fast-multiparty-ecdsa
+
+  - title: Work on schnorr multisignature schemes that require reduced interactivity
+    url: /en/newsletters/2019/11/27/#schnorr-taproot-updates
+
+  - title: Talk about how taproot enables scaling when used with multisignatures
+    url: /en/newsletters/2019/09/18/#blockchain-design-patterns-layers-and-scaling-approaches
+
+## Optional.  Same format as "primary_sources" above
+see_also:
```

Comment:
Maybe see also adapator signatures, since multiparty ECDSA and ECDSA adaptor signatures use similar techniques.